packages/lesswrong/components/posts/TableOfContents/AnswerTocRow.jsx

```diff
@@ -47,15 +69,14 @@ const AnswerTocRow = ({classes, answer}) => {
       </div>
       <div dangerouslySetInnerHTML={{__html:highlight}} className={classes.tooltip} />
     </div>
-
   return <div>
     <Tooltip title={tooltip} placement="right-start">
       <span className={classes.root}>
         <span className={classes.karma}>
           {answer.baseScore}
         </span>
-        <span>
-          {answer.author}
+        <span className={classes.firstLine}>
+          <span className={classes.author}>{answer.author}:</span> { singleLineHighlight }...
```

Comment:
Putting '...' on the end here isn't quite right; there are two cases in which it fails. The first is if the answer is very short, so that it effectively isn't abridged at all. And the second is where truncation happened at a sentence boundary, so there was already a period; then you end up with four periods, which is weird.

Reply:
(The it-wasn't-truncated case would've be handled by making truncatise append the suffix; in this case truncatise can't handle everything because there's both a word limit and a character limit, so instead compare lengths to see if anything was in fact truncated. The four-dots thing is probably a bug we have in other places, and isn't a huge deal, it was just more noticeable because of the dots on very-short strings.)